src/layout/SigningStatusPanel/SigningStatusPanelComponent.tsx

```diff
@@ -0,0 +1,88 @@
+import React from 'react';
+import { useParams } from 'react-router-dom';
+
+import { Spinner } from '@digdir/designsystemet-react';
+import { useQuery } from '@tanstack/react-query';
+
+import { Panel } from 'src/app-components/Panel/Panel';
+import { useIsAuthorised, useTaskTypeFromBackend } from 'src/features/instance/ProcessContext';
+import { Lang } from 'src/features/language/Lang';
+import { useLanguage } from 'src/features/language/useLanguage';
+import { signeeListQuery } from 'src/layout/SigneeList/api';
+import { AwaitingCurrentUserSignaturePanel } from 'src/layout/SigningStatusPanel/PanelAwaitingCurrentUserSignature';
+import { AwaitingOtherSignaturesPanel } from 'src/layout/SigningStatusPanel/PanelAwaitingOtherSignatures';
+import { NoActionRequiredPanel } from 'src/layout/SigningStatusPanel/PanelNoActionRequired';
+import { SubmitPanel } from 'src/layout/SigningStatusPanel/PanelSubmit';
+import classes from 'src/layout/SigningStatusPanel/SigningStatusPanel.module.css';
+import { ProcessTaskType } from 'src/types';
+import type { PropsFromGenericComponent } from 'src/layout';
+import type { SigneeState } from 'src/layout/SigneeList/api';
+
+export function SigningStatusPanelComponent({ node }: PropsFromGenericComponent<'SigningStatusPanel'>) {
+  const taskType = useTaskTypeFromBackend();
+  const { partyId, instanceGuid } = useParams();
+  const { data: signeeList, isLoading } = useQuery(signeeListQuery(partyId!, instanceGuid!));
+  const currentUserStatus = getCurrentUserStatus(signeeList, partyId);
+  const canWrite = useIsAuthorised()('write');
+  const { langAsString } = useLanguage();
+
+  if (taskType !== ProcessTaskType.Signing) {
+    return (
+      <Lang
+        id='signing.wrong_task_error'
+        params={['SigningStatusPanel']}
+      />
+    );
```

Comment:
Same thing here, should use component validation. Take inspiration from `Subform`, the `renderLayoutValidators()` function and `<SubformValidator />` - I think that's the currently easiest approach to it.